Wikipedia/Code/WikipediaLookup.swift

```diff
@@ -45,14 +45,14 @@ import CocoaLumberjackSwift
             let entries = try JSONDecoder().decode([String : [WikipediaLanguageVariant]].self, from: data)
             return entries.mapValues { wikipediaLanguageVariants -> [MWKLanguageLink] in
                 wikipediaLanguageVariants.map { wikipediaLanguageVariant in
-                    
+ 
                     var localizedName = wikipediaLanguageVariant.localName
                     if !Locale.current.isEnglish,
                         let iOSLocalizedName = Locale.current.localizedString(forIdentifier: wikipediaLanguageVariant.languageVariantCode) {
                         localizedName = iOSLocalizedName
                     }
                     
-                    return MWKLanguageLink(languageCode: wikipediaLanguageVariant.languageCode, pageTitleText: "", name: wikipediaLanguageVariant.languageName, localizedName: localizedName, languageVariantCode: wikipediaLanguageVariant.languageVariantCode)
+                    return MWKLanguageLink(languageCode: wikipediaLanguageVariant.languageCode, pageTitleText: "", name: wikipediaLanguageVariant.languageName, localizedName: localizedName, languageVariantCode: wikipediaLanguageVariant.languageVariantCode, altISOCode: nil)
```

Comment:
I realize we don't need an altcode for any language variants currently, but is there any harm in providing a non-nil property here (in case we get any in the future)?

Reply:
This is done, let me know if isn't what you had in mind.

Reply:
Thanks!